### ISSUE TITLE
Problem accessibility Teamtalk v 5.8.3.5062 (Windows)

### ISSUE THREAD

**Issue by math65** [BearWare/TeamTalk5#1149]
Hi,

I noticed some issues with Windows Beta and NVDA screen reader:

* When writing a private message, NVDA announces the number of characters entered. Would it be possible to have an option to disable it, or to trigger this option as soon as we get close to the character limit?
* When in the compose window of a private message, NVDA does not announce when the user is writing. Is this normal or a bug? For this to work I have to go to the desktop or to another window.

Thank you for making this app accessible!

**Comment by Oreonan** [comment]
Hi,

* This will be reimplemented in next beta, number of characters will be 
not announce anymore, in place, an alert will be emmited when limit is 
exceeded and no message will be send.

* Can you confirm that in "Preferences/Text-to-Speech events", the event 
"User is typing a private text message in focused window" is unchecked 
and "User is typing a private text message" is checked? If yes invert it.

**Comment by math65** [comment]
It's work, Thank !!! :)
